starknet_py/abi/schemas.py

```diff
@@ -27,6 +27,7 @@ class FunctionBaseSchema(Schema):
     outputs = fields.List(
         fields.Nested(TypedParameterSchema()), data_key="outputs", required=True
     )
+    state_mutability = fields.String(data_key="stateMutability", required=False)
```

Comment:
Wouldn't that cause issues when trying to construct an ABI object from the result of this schema? A dict used to construct a dataclass would have field `state_mutability` that is not declared in the dataclass itself. Perhaps this should be addressed by adding `unknown=EXCLUDE` somewhere around `ContractClassSchema`. 

Btw I don't see the `stateMutability` declared anywhere in the rpc spec 0.2.1 https://github.com/starkware-libs/starknet-specs/blob/v0.2.1/api/starknet_api_openrpc.json or on the master branch. This seems more like an issue with non compliant rpc provider than our handling.

Reply:
I can look into tracking down the right spot for the unknown=EXCLUDE, I dont know the status or background on the state_mutability field but [this spec PR](https://github.com/starkware-libs/starknet-specs/pull/60) has been open since Nov

Reply:
Switched to EXCLUDE